ext/opentelemetry-ext-docker-tests/tests/check_availability.py

```diff
@@ -33,76 +34,82 @@
 POSTGRES_PASSWORD = os.getenv("POSTGRESQL_HOST", "testpassword")
 POSTGRES_PORT = int(os.getenv("POSTGRESQL_PORT", "5432"))
 POSTGRES_USER = os.getenv("POSTGRESQL_HOST", "testuser")
+REDIS_HOST = os.getenv("REDIS_HOST", "localhost")
+REDIS_PORT = int(os.getenv("REDIS_PORT ", "6379"))
 RETRY_COUNT = 5
 RETRY_INTERVAL = 5  # Seconds
 
 logger = logging.getLogger(__name__)
 
 
+def retryable(func):
+    def wrapper():
+        # Try to connect to DB
+        for i in range(RETRY_COUNT):
+            try:
+                func()
+                return
+            except Exception as ex:  # pylint: disable=broad-except
+                logger.error(
+                    "waiting for %s, retry %d/%d [%s]",
+                    func.__name__,
+                    i + 1,
+                    RETRY_COUNT,
+                    ex,
+                )
+            time.sleep(RETRY_INTERVAL)
+        raise Exception("waiting for %s failed")
```

Comment:
I think a string argument to replace `%s` is missing.

Reply:
Good catch. Fixed